modules/ROOT/Edits/configure-third-party.adoc

```diff
@@ -0,0 +1,21 @@
+= Configure and Manage 3rd Party Connections
+
+If your trading partners use 3rd party service providers such as VANs or 3PLs (3rd party logistics providers) who manage the B2B connectivity with your organization on behalf of your partners, you can create a 3rd party connection to manage configurations such as the endpoints, identifiers, and certificates of the 3rd party provider.
+
+After you create a 3rd party connection, you can select the endpoints and identifiers configured for that connection in message flows created for your partners.
+
+Use the *Partners* page to create and manage 3rd party connections. From this page, you can:
```

Comment:
Does it bother you that the prev. para is about after creating and this one is about creating? I'm probably being too picky, but I felt a...disjoint.